### DISCUSSION TITLE
FYI Lesson 11 -curses

### DISCUSSION THREAD

**Discussion by Callassie** [smartcontractkit/full-blockchain-solidity-course-py#1376] [General]
I had to install curses..

python -m pip install windows-curses

in order for the metadata portion to work for lesson 10.

Did a search and did not see any previous discussion regarding this

**Comment by cromewar** [comment]
Hello @Callassie 
This is interesting, could you provide more info on this? on which context you have to install it and what was the problem you faced in order to need it?

**Comment by Callassie** [comment]
It was Lesson 11- not 10. Edited title
When writing create_metadata and started using Path to create the binary. I have Python 3.10 and it just didn't have curses.  Very easy fix once I searched it.
Maybe this makes more sense being that I had the lesson wrong. Sorry!

**Reply by cromewar** [reply]
Oh interesting, as I'm developing using WSL (LInux for Windows) and Patrick used MacOS I think neither of us encountered that problem, I'll add that as an annotation for the Spanish version of the course.
Thanks.

**Reply by Callassie** [reply]
Maybe it will help someone.
